ansible/playbooks/roles/repository/tasks/copy-download-requirements.yml

```diff
@@ -50,3 +51,15 @@
         src: "download-requirements/{{ item }}"
         dest: "{{ download_requirements_dir }}/{{ item }}"
       loop: "{{ _files }}"
+
+    - name: Get stats of optional repo files
+      stat:
+        path: "download-requirements/{{ _repo_files }}"
```

Comment:
Does it work when `_repo_files` is a list?

Reply:
by repo_files we mean requirements/{{ architecture }}/{{ os_family }}/files.yml